### ISSUE TITLE
data heatmap data issue

### ISSUE THREAD

**Issue by haishiro** [data-describe/data-describe#411]
Your bug may already be reported!
Please search on the [issue tracker](../issues?q=label%3Abug+is%3Aissue) before creating one.

**Description**

mths_since_last_record has 91% nulls
![image](https://user-images.githubusercontent.com/11744859/100964047-64533400-34ed-11eb-90cc-f85c5bbd57be.png)

This is different using the plotly backend:
![image](https://user-images.githubusercontent.com/11744859/100964097-81880280-34ed-11eb-82f5-54fe8c033985.png)

**Steps to Reproduce**

```df = pd.read_csv("gs://amazing-public-data/lending_club/lending_club_data.tsv", sep="\t")```

**Your Environment**
* data describe version:
* Python version:
* Operating System:

**Comment by truongc2** [comment]
@haishiro Is the assumption that missing values are marked as white? So in this case, the plotly plot is correct and the seaborn plot is the incorrect version?

**Comment by haishiro** [comment]
Yes, should be white. `mths_since_last_record` should have a very high percentage of missing values so it should be almost all white like the plotly plot.
